src/Components/Nav/FloatingBtn.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styled from "styled-components";
+
+function FloatingBtn(props) {
+  return (
+    <Wrapper onClick={props.scrollToTop}>
```

Comment:
```suggestion
function FloatingBtn({ scrollToTop }) {
  return (
    <Wrapper onClick={scrollToTop}>
```
파라미터 자리에서 직접 구조분해 할당 해주세요

Reply:
수정 완료